Serializer/JMSSerializerAdapter.php

```diff
@@ -34,10 +34,18 @@ class JMSSerializerAdapter implements Serializer
     const DESERIALIZATION = 1;
 
     private $serializer;
+    private $serializationContextFactory;
+    private $deserializationContextFactory;
 
-    public function __construct(SerializerInterface $serializer)
+    public function __construct(
+        SerializerInterface $serializer,
+        SerializationContextFactoryInterface $serializationContextFactory,
```

Comment:
Oh indeed, I missed that. We could default the argument to `null` and fall back to the old code then, right?